web/controllers/page_controller.ex

```diff
@@ -8,4 +8,8 @@ defmodule Webapp.PageController do
   def login(conn, _params) do
     render conn, "index.html"
   end
+
+  def docs(conn, _params) do
+    render conn, "docs.html"
```

Comment:
Please use **index.html** as the template. All the React stuff is rendered in a single page which currently is the mentioned template and as you can see in the code, it has a DOM element with id="react-root"